client/src/main.ts

```diff
@@ -0,0 +1,15 @@
+import Vue from 'vue'
+import App from './App.vue'
+import router from './router'
+import store from './store'
+import vuetify from './plugins/vuetify';
+import 'material-design-icons-iconfont/dist/material-design-icons.css'
+
+Vue.config.productionTip = false
```

Comment:
You might also want to add
`// get rid of "Download the Vue Devtools extension for a better development experience" console message
Vue.config.devtools = false`.